databricks/koalas/utils.py

```diff
@@ -411,6 +411,50 @@ def validate_axis(axis=0, none_axis=0):
     return {None: none_axis, 'index': 0, 'columns': 1}.get(axis, axis)
 
 
+def temp_column_name(sdf: spark.DataFrame, gen: Union[str, Callable[[int], str]]) -> str:
```

Comment:
@ueshin do you plan to use `temp_column_name(sdf, '{}'.format)` style very soon? Seems currently not being used, and might be best to just remove.

Reply:
ah, yeah, it's from the original idea and I don't use it anymore. We can remove it.